apps/api/src/billing/services/wallet-initializer/wallet-initializer.service.ts

```diff
@@ -2,14 +2,20 @@ import { singleton } from "tsyringe";
 
 import { AuthService } from "@src/auth/services/auth.service";
 import { UserWalletInput, UserWalletPublicOutput, UserWalletRepository } from "@src/billing/repositories";
+import { FeatureFlags } from "@src/core/services/feature-flags/feature-flags";
+import { FeatureFlagsService } from "@src/core/services/feature-flags/feature-flags.service";
+import { NotificationService } from "@src/notifications/services/notification/notification.service";
+import { startTrial } from "@src/notifications/services/notification-templates/start-trial";
```

Comment:
suggestion(non-blocking): perhaps we can rename this to `startTrialTemplate` to be more specific about what it does